src/vote-subjects/entities/vote-subject.entity.ts

```diff
@@ -9,29 +9,29 @@ import {
 @Entity()
 export class VoteSubject {
   @PrimaryGeneratedColumn()
-  id!: number;
+  id!: string;
 
   @Column({ nullable: false, length: 100 })
   title!: string;
 
-  @CreateDateColumn({ nullable: false })
-  created!: Date;
+  @CreateDateColumn({ name: 'created_at', nullable: false })
+  createdAt!: Date;
 
-  @UpdateDateColumn({ nullable: true, default: null })
-  updated?: Date;
+  @UpdateDateColumn({ name: 'updated_at', nullable: true, default: null })
+  updatedAt?: Date;
```

Comment:
```suggestion
  updatedAt!: Date | null;
```
@sua-no `null`이 기본값이므로 optional 대신 명시하는게 맞을 것 같은데, 의견 주시면 감사하겠습니다.

Reply:
@sua-no 생각해보니 이 컬럼 특징은 `deletedAt`인 것 같고, `updatedAt`은 NotNull에 기본값은 `createdAt`일 것 같은데 어떠실까요?

Reply:
seed넣어보니까 알아서 updatedAt에 기본값 createdAt이네요 
notNull로 변경하겠습니다